process/block/interceptedBlocks/interceptedBlockHeader_test.go

```diff
@@ -0,0 +1,349 @@
+package interceptedBlocks_test
+
+import (
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	block2 "github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/process/block/interceptedBlocks"
+	"github.com/ElrondNetwork/elrond-go/process/mock"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+	"github.com/stretchr/testify/assert"
+)
+
+var testMarshalizer = &mock.MarshalizerMock{}
+var testHasher = mock.HasherMock{}
+var hdrNonce = uint64(56)
+var hdrShardId = uint32(1)
+var hdrRound = uint64(67)
+var hdrEpoch = uint32(78)
+
+func createDefaultArgument() *interceptedBlocks.ArgInterceptedBlockHeader {
+	arg := &interceptedBlocks.ArgInterceptedBlockHeader{
+		ShardCoordinator: mock.NewOneShardCoordinatorMock(),
+		MultiSigVerifier: mock.NewMultiSigner(),
+		Hasher:           testHasher,
+		Marshalizer:      testMarshalizer,
+		ChronologyValidator: &mock.ChronologyValidatorStub{
+			ValidateReceivedBlockCalled: func(shardID uint32, epoch uint32, nonce uint64, round uint64) error {
+				return nil
+			},
+		},
+	}
+
+	hdr := createMockHeader()
+	arg.HdrBuff, _ = testMarshalizer.Marshal(hdr)
+
+	return arg
+}
+
+func createMockHeader() *block2.Header {
+	return &block2.Header{
+		Nonce:            hdrNonce,
+		PrevHash:         []byte("prev hash"),
+		PrevRandSeed:     []byte("prev rand seed"),
+		RandSeed:         []byte("rand seed"),
+		PubKeysBitmap:    []byte("bitmap"),
+		ShardId:          hdrShardId,
+		TimeStamp:        0,
+		Round:            hdrRound,
+		Epoch:            hdrEpoch,
+		BlockBodyType:    block2.TxBlock,
+		Signature:        []byte("signature"),
+		MiniBlockHeaders: nil,
+		PeerChanges:      nil,
+		RootHash:         []byte("root hash"),
+		MetaBlockHashes:  nil,
+		TxCount:          0,
+	}
+}
+
+//------- TestNewInterceptedHeader
```

Comment:
delete comment.

Reply:
I would rather keep a limited number of placeholders only in test files as all tests start with Test... and is harder to keep the tests grouped by the tested func.